src/main/java/dynamicfps/mixin/GameRendererMixin.java

```diff
@@ -23,7 +22,7 @@
 	 Implements the mod's big feature.
 	 */
 	@Inject(at = @At("HEAD"), method = "render", cancellable = true)
-	private void onRender(CallbackInfo callbackInfo) {
+	private void onRender(float tickDelta, long limitTime, boolean tick, CallbackInfo callbackInfo) {
```

Comment:
Interesting, so mixin requires you to specify the arguments now?

Reply:
Seems like it - without arguments the mixin couldn’t be applied 🤷

Reply:
Huh, interesting. I'll have to check for myself, but that's good to know!